hack/goveralls.sh

```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env bash
+set -eo pipefail
 
-set -e
-
-./hack/coverage.sh
-goveralls -v -debug -service=${CI_NAME} -coverprofile=.coverprofile -ignore=$(find -regextype posix-egrep -regex ".*generated_mock.*\.go|.*swagger_generated\.go|.*openapi_generated\.go" -printf "%P\n" | paste -d, -s)
+bazel run //vendor/github.com/wadey/gocovmerge:gocovmerge -- $(cat | sed "s# # ${BUILD_WORKING_DIRECTORY}/#g" | sed "s#^#${BUILD_WORKING_DIRECTORY}/#") >coverprofile.dat
```

Comment:
Wow, never seen a subshell consume stdin from its callers caller, but yeah that works :)
```
$ echo test | echo $( cat )
test
$
```